core/rs/core/src/sha.rs

```diff
@@ -1,2 +1,2 @@
 // The sha of the commit that this version of crsqlite was built from.
-pub const SHA: &'static str = "";
+pub const SHA: &'static str = "3a01980562615001d765eec61e3ed58147a93f93";
```

Comment:
hm, I guess I should ignore this file and only generate it at build time?

Reply:
Oh, possibly yes. I didn't notice it.